docs/adv-topics/sip-phone.md

```diff
@@ -69,7 +69,7 @@ chan_pjsip requires a number of modules to be loaded.  You should start by editi
 
 ## Update extensions.conf
 
-The next step is to update extension.conf.  This creates a dial plan that controls access to your node by sip phones.  Edit /etc/asterisk/extensions.conf and add the following to the bottom of the file.  This example will set up extension 1001 as your local phone and the dialing plan for your node.
+The next step is to update extension.conf.  This creates a dial plan that controls access to your node by sip phones.  Edit /etc/asterisk/extensions.conf and add the following to the bottom of the file.  This example will set up extension 1001 as your local phone and the dialing plan for your node. 
```

Comment:
Why the added space at the end of the line?

Also, should we change "sip" to "SIP"?